client/app/scripts/superdesk-archive/directives.js

```diff
@@ -396,6 +396,21 @@
                 }
             };
         }])
+        .directive('sdDraggableItem', function() {
+            return {
+                link: function(scope, elem) {
+                    if (scope.item) {
```

Comment:
it's used on element with `ng-repeat="item in ..."` so not sure it can have isolated scope